roofit/roofitcore/inc/RooAbsReal.h

```diff
@@ -389,9 +389,8 @@ class RooAbsReal : public RooAbsArg {
 
 
  public:
-  const RooAbsReal* createPlotProjection(const RooArgSet& depVars, const RooArgSet& projVars, RooArgSet*& cloneSet) const ;
-  const RooAbsReal *createPlotProjection(const RooArgSet &dependentVars, const RooArgSet *projectedVars,
-                     RooArgSet *&cloneSet, const char* rangeName=nullptr, const RooArgSet* condObs=nullptr) const;
+  std::unique_ptr<RooAbsReal> createPlotProjection(const RooArgSet &dependentVars, const RooArgSet *projectedVars,
```

Comment:
This is a change of a public interface. Is it really needed ?

Reply:
No it's not. First I thought the support for the batchMode would require rewriting `createPlotProjection`, that's why I touched it, but it turned out that was not necessary.

I'll just drop the first commit then